### PR TITLE
Added Landing Page for Component Library.

### PR DESCRIPTION
**Pull request by vijaytembugade**
Added responsive Landing page for documentation side.
Live preview: https://vijaytembugade.github.io/XenonUI-Component-Library/

image

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  flex-wrap: wrap;
+}
+.main-logo {
+  font-size: 7rem;
+  font-weight: bold;
+  padding: 1rem;
+}
+
+.main-logo::first-letter {
+  color: #9370db;
```
Comment (srishti-maurya) [inline index.css]: maybe you may use color variables here

**Review by srishti-maurya — APPROVED** [review]
looks good

File: index.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    font-size: 4rem;
+  }
+  .main-logo::first-letter {
+    color: #9370db;
+    font-size: 5rem;
+  }
+  .design-text {
+    color: #f0739d;
+    font-size: 5rem;
+  }
```
Comment (kshitijthatte) [inline index.css]: color is already defined for these properties, outside the media query.

**Review by kshitijthatte — APPROVED** [review]

**Review by techieshubham30 — APPROVED** [review]

File: index.css (hunk 1 above)
Comment (vijaytembugade) [reply index.css]: Thank for pointing it out srishti. I will work on it.

File: index.css (hunk 2 above)
Comment (vijaytembugade) [reply index.css]: Yes. My bad. I will remove them.

**Review by horsemaker — APPROVED** [review]

**Comment by AmitNemade** [comment]
color selection is too good
Looking for more components
@vijaytembugade

**Pull request merged**